### PR TITLE
docs: improve the english doc of maintenance - operations in cli

### PR DESCRIPTION
**Pull request by TanZiYen**

### CODE REVIEW

File: docs/en/maintain/cli.md [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
 ## NS Client
 
-Connecting to the NS Client requires specifying zk\_cluster, zk\_root\_path and role. Where zk\_cluster is the zk address, zk\_root\_path is the root path of the cluster in zk, role is the role to be started and needs to be specified as ns_client
+Connecting to the NS Client requires specifying zk\_cluster, zk\_root\_path and role. Where zk\_cluster is the zk address, zk\_root\_path is the root path of the cluster in zk and role is the role to be started, and needs to be specified as ns_client
 
 ```bash
 $ ./bin/openmldb --zk_cluster=172.27.2.52:12200 --zk_root_path=/onebox --role=ns_client
 ```
 
-### use
```
Comment (lumianph) [inline docs/en/maintain/cli.md]: those starting with ### (including the below) are commands, no need to capitalize them, please revert them to the lower cases

File: docs/en/maintain/cli.md [hunk 2]
```diff
@@ -131,19 +131,19 @@ DelReplica ok
...
 
 Replicas migration
 
 Command format: `migrate src_endpoint table_name pid_group des_endpoint`
 
 * src\_endpoint: The endpoint of the node that needs to be checked out
-* table\_name: the table name
-* pid\_group:  the collection of shard IDs. There can be the following situations
-    * a single shard
+* table\_name: The table name
```
Comment (lumianph) [inline docs/en/maintain/cli.md]: this is the description for a specific term, by convention we also no need to capitalize the first word, please help change them back.

**Review by lumianph — CHANGES_REQUESTED** [review]
pls fix the capitalization for commands and terms

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by lumianph — APPROVED** [review]

**Pull request merged**